dynamic_rest/viewsets.py

```diff
@@ -232,6 +232,26 @@ def get_request_fields(self):
         self._request_fields = request_fields
         return request_fields
 
+    def get_request_debug(self):
+        debug = self.get_request_feature(self.DEBUG)
+        if debug:
+            debug = debug .lower()
+            if debug == 'true':
+                debug = True
+            else:
+                debug = False
+        return debug
+
+    def get_request_sideloading(self):
+        sideloading = self.get_request_feature(self.SIDELOADING)
+        if sideloading:
+            sideloading = sideloading.lower()
+            if sideloading == 'true':
```

Comment:
Might be nice to reuse [this list](https://github.com/AltSchool/dynamic-rest/blob/master/dynamic_rest/filters.py#L179) for consistent handling of boolean values.